lib/db.js

```diff
@@ -1,3 +1,4 @@
+/// <reference path="types.js" />
```

Comment:
Q: I have not seen this before with jsdocs, what is it doing?

Reply:
its to reference types defined in another file. like a `require` for types.

Not sure if this is the best way though.

Reply:
Interesting, I had no idea importing them was actually required!